tests/ami/scripts/fresh_install_deb.sh.j2

```diff
@@ -28,20 +28,20 @@ cat /etc/*-release
 sudo apt-get update -y
 sudo apt-get install -y "{{python_package}}"
 
-{% if not package_to_version_is_url -%}
+# Install provided version
+{% if package_to_version_source_type == "install_script" -%}
 wget -O "install-scalyr-agent-2.sh" -q "{{installer_script_url}}"
 sudo bash ./install-scalyr-agent-2.sh --set-api-key "{{scalyr_api_key}}"
-{% endif -%}
+sudo apt-get install -y "scalyr-agent-2={{package_to_version}}"
+export PACKAGE_TO_VERSION="{{package_to_version}}"
+{% elif package_to_version_source_type in ("file", "url") -%}
```

Comment:
Perhaps it would be a good idea to add a comment if the source is file that the file is uploaded by the script / Libcloud itself.

Reply:
Yeah, good idea.